testing/adios2/engine/bp/TestBPWriteReadSCR.cpp

```diff
@@ -0,0 +1,1044 @@
+/*
+ * Distributed under the OSI-approved Apache License, Version 2.0.  See
+ * accompanying file Copyright.txt for details.
+ */
+#include <cstdint>
+#include <cstring>
+
+#include <iostream>
+#include <numeric> //std::iota
+#include <stdexcept>
+
+#include <adios2.h>
+
+#include <gtest/gtest.h>
+
+#include "../SmallTestData.h"
+
+#include "scr.h"
```

Comment:
Similarly here `#include <scr.h>` would help identify it as an external header. Thanks for adding a test, BTW.